Sources/Private/Model/DotLottie/ZipFoundation/README.md

```diff
@@ -0,0 +1,24 @@
+## ZipFoundation
```

Comment:
Added some more discussion about our ZIPFoundation dependency, plus instructions for if we have to update this again in the future.